src/seedsigner/models/settings_definition.py

```diff
@@ -53,28 +58,34 @@ class SettingsConstants:
     LOCALE__RUSSIAN = "ru"
     LOCALE__SPANISH = "es"
     ALL_LOCALES = [
-        (LOCALE__ARABIC, "Arabic"),
-        (LOCALE__CZECH, "Czech"),
+        # TRANSLATOR_NOTE: Translator is expected ONLY to change their own locale below
+        (LOCALE__ARABIC, _("Arabic")),
+        (LOCALE__CZECH, _("Czech")),
         (LOCALE__ENGLISH, "English"),
-        (LOCALE__FRENCH, "French"),
-        (LOCALE__GERMAN, "German"),
-        (LOCALE__HEBREW, "Hebrew"),
-        (LOCALE__JAPANESE, "Japanese"),
-        (LOCALE__KOREAN, "Korean"),
-        (LOCALE__PORTUGUESE, "Portuguese"),
-        (LOCALE__RUSSIAN, "Russian"),
-        (LOCALE__SPANISH, "Spanish"),
+        (LOCALE__FRENCH, _("French")),
+        (LOCALE__GERMAN, _("German")),
+        (LOCALE__HEBREW, _("Hebrew")),
+        (LOCALE__JAPANESE, _("Japanese")),
+        (LOCALE__KOREAN, _("Korean")),
+        (LOCALE__PORTUGUESE, _("Portuguese")),
+        (LOCALE__RUSSIAN, _("Russian")),
+        (LOCALE__SPANISH, _("Spanish")),
     ]
+    # Some locales disabled below
+    ALL_LOCALES.remove((LOCALE__ARABIC, _("Arabic")))
```

Comment:
Simpler to just comment out lines 62, 67, 68, 69 above?

Reply:
I believe so.

I probably started by deleting the lines instead of commenting them out, then realized I couldn't remember what was there and wanted to make it more explicit that all langs are intended... but in my temporary exploration they are being removed.